### PR TITLE
fix: Prevent new lines in My Profile bio

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Bugfix**

This PR resolves [CX-2321]

### Description

This PR fixes an issue where it was possible to add many new lines to the bio which then makes the My Collection page unusable (see screenshot).

Screenshot 2022-02-04 at 15 51 14

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Prevent new lines in My Profile bio - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2321]: https://artsyproduct.atlassian.net/browse/CX-2321?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 42b87ed434d5fbe4456ceb559d76e97e5f262b03
</p>

**Review by MounirDhahri — COMMENTED** [review]
✨

**Review by MounirDhahri — APPROVED** [review]

**Review by MrSltun — APPROVED** [review]
🚀

File: src/lib/Scenes/MyProfile/MyProfileEditFormModal.tsx [hunk 1]
```diff
@@ -344,13 +345,14 @@ export const MyProfileEditFormModal: React.FC<MyProfileEditFormModalProps> = ({
                   <Input
                     ref={bioInputRef}
                     title="About"
-                    onChangeText={handleChange("bio")}
+                    onChangeText={(text) => setFieldValue("bio", normalizeMyProfileBio(text))}
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/lib/Scenes/MyProfile/MyProfileEditFormModal.tsx:348]: Good point. I decided to do it here so users are not able to write paragraphs that then get removed afterward. But let's have a look with the team to discuss what the best solution would be.

**Pull request merged**